### PR TITLE
Ht18 grant

### PR DESCRIPTION
**Pull request by mykolamuntian**
To create a program for the accrual of scholarships to students on the results of the examination session. Information on the results of the session includes:
- last name;
- name;
- patronymic;
- group number;
- examination marks.
The number of examination marks is at least 3 and not more than 5. The scholarship is awarded to students who passed all the exams in the session, according to the following rules. Students who passed all the exams for "excellent" receive a mark-up equal to 100%; students who passed the exams for "good" and "excellent" - 50%; and students who passed exams for "good" - 25%. The scholarship is not awarded to students who have more than two satisfactory grades in the session.
List of students of each group that received a scholarship, display on the screen, ordering it alphabetically.

### CODE REVIEW

File: src/com/muntian/ht18_Grant/AccrualOfScholarships.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    private static boolean compareStudentsByLastName(Student student1, Student student2) {
+        return student1.getLastName().compareTo(student2.getLastName()) > 0;
+    }
+
+    private static void printStudents(Student[] listOfStudents) {
+        System.out.println("Last name\t\t\tFirst name\t\tAn increase to a scholarship");
+        System.out.println("-----------------------------------------------------------------");
+        for (int i = 0; i < listOfStudents.length; ++i) {
+            switch (listOfStudents[i].getAcademicPerformance()) {
+                case 5:
```
Comment (strazhkoe) [inline src/com/muntian/ht18_Grant/AccrualOfScholarships.java]: Эти числа из кейсов нужно вынести в константы

File: src/com/muntian/ht18_Grant/Student.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+//        for (int i = 0; i<marks.length;i++){
+//            if (marks[i]!=0){
+//                numberOfTests++;
+//            }
+//        }
+//        return numberOfTests;
+//    }
+
+    public int getAcademicPerformance() {
+        return academicPerformance;
```
Comment (strazhkoe) [inline src/com/muntian/ht18_Grant/Student.java]: Тут должно происходить вычисление исходя из оценок

File: src/com/muntian/ht18_Grant/Student.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    }
+
+    public void setAcademicPerformance(int academicPerformance) {
```
Comment (strazhkoe) [inline src/com/muntian/ht18_Grant/Student.java]: этого метода быть не должно

**Review by strazhkoe — CHANGES_REQUESTED** [review]

**Comment by mykolamuntian** [comment]
Task was fixed according to the comments

**Review by strazhkoe — APPROVED** [review]

**Pull request merged**